tests/desktop/test_collections.py

```diff
@@ -52,24 +52,21 @@ def test_create_and_delete_collection(self, mozwebqa):
     @pytest.mark.native
     @pytest.mark.nondestructive
     @pytest.mark.login
-    def test_user_my_collections_page(self, mozwebqa):
-
+    def test_user_my_collections_page(self, mozwebqa, existing_user):
         home_page = Home(mozwebqa)
-        home_page.login()
+        home_page.login(existing_user['email'], existing_user['password'])
         Assert.true(home_page.is_the_current_page)
         Assert.true(home_page.header.is_user_logged_in)
 
-        username = mozwebqa.credentials['default']['name']
         my_collections_page = home_page.header.click_my_collections()
-        Assert.equal('Collections by %s :: Add-ons for Firefox' % username, home_page.page_title)
-        Assert.equal('Collections by %s' % username, my_collections_page.my_collections_header_text)
+        Assert.equal('Collections by %s :: Add-ons for Firefox' % existing_user['name'], home_page.page_title)
```

Comment:
Nit: This line, and the next one, are awfully long. Suggest breaking onto two lines.